toolchain/parse/tree.h

```diff
@@ -171,11 +175,52 @@ class Tree : public Printable<Tree> {
   // Verifies the parse tree structure. Checks invariants of the parse tree
   // structure and returns verification errors.
   //
-  // This is primarily intended to be used as a
-  // debugging aid. This routine doesn't directly CHECK so that it can be used
-  // within a debugger.
+  // This is fairly slow, and is primarily intended to be used as a debugging
+  // aid. This routine doesn't directly CHECK so that it can be used within a
+  // debugger.
   auto Verify() const -> ErrorOr<Success>;
```

Comment:
Could we move this below the non-verify extract APIs, but keep it with the verifying extract API?

Reply:
Done.